platformcomponents/desktop/content-list-item.json

```diff
@@ -0,0 +1,25 @@
+{
+  "content-list-item": {
+    "comment": "Applied to file attachments from Content tab",
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=49651%3A114181",
+    "iconForeground": "@theme-common-text-white",
+    "iconBackground": {
+      "default": "@theme-background-accent-normal"
+    },
+    "#normal": {
+      "background": "@theme-background-secondary-normal",
+      "textFileName": "@theme-text-primary-normal",
+      "textFileSize": "@theme-text-secondary-normal"
+    },
+    "#hovered": {
+      "background": "@theme-background-secondary-hover",
+      "textFileName": "@theme-text-primary-normal",
+      "textFileSize": "@theme-text-secondary-normal"
+    },
+    "#disabled": {
```

Comment:
`#active` is missing no?

Reply:
Pushed commit
[Added active state for background and grouped states in content-file-list-item](https://github.com/momentum-design/tokens/pull/174/commits/ced4861b369eb26fd8fa719a0b4418fc6c3f2692)
Changing this conversation to Resolved.